dist/redhat/build_rpm.sh

```diff
@@ -56,8 +56,8 @@ if [ ! -e SCYLLA-RELOCATABLE-FILE ]; then
     exit 1
 fi
 
-if [ -z "$TARGET" ] || [ "$TARGET" = "centos" ]; then
-    TARGET=centos7
+if [ -z "$TARGET" ] ; then
+    TARGET=centos
 fi
```

Comment:
Let's keep support for centos7 too so people's scripts don't break. We can just ignore TARGET completely.